_UI/_web_interface/kraken_web_interface.py

```diff
@@ -181,8 +181,9 @@ def __init__(self):
         self.daq_if_gains = "[,,,,]"
         self.en_advanced_daq_cfg = False
         self.en_basic_daq_cfg = False
-        self.en_system_control = []
-        self.en_beta_features = []
+        self.en_system_control = dsp_settings.get("en_system_control", [])
```

Comment:
@krakenrf why are this initialized to lists in a first place? Shouldn’t they simply be “False”?

Reply:
This is how Checklist works https://dash.plotly.com/dash-core-components/checklist

Reply:
Then perhaps we can switch to Checkbox instead? In any case it is a simple Boolean that we are after, so list feels as an overkill.

Reply:
> Then perhaps we can switch to Checkbox instead? In any case it is a simple Boolean that we are after, so list feels as an overkill.

@godsic I've not found the Checkbox in dash, maybe due to this Karl added Checklist component instead

Reply:
Fair enough. I would still suggest to store setting as bool and turn in into list in the corresponding callback. This would keep settings file human readable and free from dash implementation details.